README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
-
-Helpfulness also comes into play with the level of conscientiousness and proactivity used to identify and resolve problems. Our friends at CB Insights have this great [framework for levels of helpfulness](https://slack-files.com/T029BMC1C-F6HNVKC8Y-e2cfa43f7d).
-
-Of course, we all get busy from time to time, so occasionally the answer might be “Not now, but have some free time later today” or “after I ship this big feature tomorrow”.
-
 ### Seek and embrace diversity
+* Actively learn about and work to mitigate your own biases
+* Make sure our product is inclusive
+* Seek representation & inclusion (even at the expense of other goals)
+* Build inclusivity & bias mitigation into our processes
+* Expect it to be hard and uncomfortable sometimes
+
 Our industry has [well-documented](http://www.theverge.com/2015/8/20/9179853/tech-diversity-scorecard-apple-google-microsoft-facebook-intel-twitter-amazon) [problems](http://wpusa.org/WPUSA_TechsDiversityProblem.pdf) when it comes to [workplace diversity](http://techcrunch.com/2015/07/28/what-a-truly-diverse-tech-company-would-look-like/). This fact makes it easy for tech companies to skew everything from their employment policies to the features they develop toward the needs of the dominant group - which is both a disservice to our customers and reinforces the lack of diversity in our industry.
 
 We work better when we include a diversity of opinions and ideas. This means encouraging input from everyone on the team, and also seeking to add people to the team from backgrounds that are traditionally underrepresented in our industry.
 
 Our dedication to diversity goes beyond simply putting equal opportunity copy on our job descriptions. We use tools (like [Textio](https://textio.com/)) to eliminate gender bias from the language of our job postings, and actively seek out applicants from diverse backgrounds. And we invite feedback from job applicants and employees alike about how we can better embrace diversity. We all have [unconscious biases](https://www.youtube.com/watch?v=nLjFTHTgEVU), and welcome your help in identifying and mitigating them.
 
 ### Delight the customer
+* Provide a delightful experience
+* Make it fun & easy
+* Provide value over the long term
+* Take time to listen to and empathize with the customer
+* Be accountable for how people experience our product and company
+
 This is a value as old as business itself, but it’s fundamental to how we operate at Bonusly. We are a business, and the only way we can stay in business is by attracting and retaining customers.
 
 At Bonusly, everyone focuses on the customer -- their needs, challenges, feedback, ideas, and behaviors. This focus is foundational to everything we do, because we don’t just serve customers -- we delight them.
```

Comment:
I liked Betty's idea of emphasizing success -- I think it would be great to say something in this description about helping customers find success. 

I also really liked the comments from the discussion about people liking this value because it means that we build something that people actually use and enjoy. Could be another detail to add in.